calendar-bundle/src/EventListener/PreviewUrlCreateListener.php

```diff
@@ -61,7 +61,7 @@ public function __invoke(PreviewUrlCreateEvent $event): void
             return;
         }
 
-        if (null === ($eventModel = $this->getEventModel($this->getId($event, $request)))) {
+        if ((!$id = $this->getId($event, $request)) || (!$eventModel = $this->getEventModel($id))) {
```

Comment:
```suggestion
        if (!($id = $this->getId($event, $request)) || !($eventModel = $this->getEventModel($id))) {
```
I think our coding style is to put the exclamation mark before the braces. (But I’m not 100% sure).

Reply:
No, it depends.

```php
// Before the braces if the ! denies more than just what's inside the braces (it refers to instanceof in this case)
if (!($foo = $this->getFooPage()) instance of PageModel)

// Before the braces if the ! denies two or more conditions inside the braces
if (!(foo() && bar()))

// But inside the braces if the ! denies a variable assignment
if ((!$id = $this->getId()) || foo())
```

Reply:
K, sorry for the interruption ☺️